saleor/graphql/account/mutations/staff.py

```diff
@@ -156,30 +180,74 @@ class Meta:
         exclude = ["password"]
         model = models.User
         permissions = (AccountPermissions.MANAGE_STAFF,)
-        error_type_class = AccountError
-        error_type_field = "account_errors"
+        error_type_class = StaffError
+        error_type_field = "staff_errors"
 
     @classmethod
     def clean_input(cls, info, instance, data):
         cleaned_input = super().clean_input(info, instance, data)
 
+        errors = defaultdict(list)
         if cleaned_input.get("redirect_url"):
             try:
                 validate_storefront_url(cleaned_input.get("redirect_url"))
             except ValidationError as error:
-                raise ValidationError(
-                    {"redirect_url": error}, code=AccountErrorCode.INVALID
-                )
+                error.code = AccountErrorCode.INVALID
```

Comment:
Not sure if we shouldn't introduce `StaffError` enum here? I guess we should discuss that with the team.

Reply:
If we agree that it's a good idea then let's change it (I see two thumbs-ups 😉).

Reply:
After the team discussion, we probably should leave it as it is.